65.md

```diff
@@ -4,73 +4,56 @@ NIP-65
+## When to Use Read and Write
 
-Clients SHOULD read events that tag their user from their user's read relays.
+When seeking events **from** a user, Clients SHOULD use the WRITE relays of the user's `kind:10002`
 
-Clients SHOULD write events that tag a person to at least some of that person's read relays. Explicitly, they SHOULD NOT expect that person will pick them up from their user's write relays. It SHOULD NOT be presumed that the user's write relays coincide with the read relays of the person being tagged.
+When seeking events **about** a user, where the user was tagged, Clients SHOULD use the READ relays of the user's `kind:10002` 
 
-Clients SHOULD presume that if their user has a pubkey in their ContactList (kind 3) that it is because they wish to see that author's feed-related events. But clients MAY presume otherwise.
+When broadcasting an event, Clients SHOULD:
 
-### Motivation
+- Broadcast the event to the WRITE relays of the author
+- Broadcast the event all READ relays of each tagged user. 
 
-There is a common nostr use case where users wish to follow the content produced by other users. This is evidenced by the implicit meaning of the Contact List in [NIP-02](02.md)
+## Motivation
 
-Because users don't often share the same sets of relays, ad-hoc solutions have arisen to get that content, but these solutions negatively impact scalability and decentralization:
+The old model of using a fixed relay list per user centralizes in large relay operators: 
 
-  - Most people are sending their posts to the same most popular relays in order to be more widely seen
-  - Many people are pulling from a large number of relays (including many duplicate events) in order to get more data
+  - Most users submit their posts to the same highly popular relays, aiming to achieve greater visibility among a broader audience.
+  - Many users are pulling events from a large number of relays in order to get more data at the expense of duplication
   - Events are being copied between relays, oftentimes to many different relays
+  
+This NIP allows Clients to connect directly with the most up-to-date relay set from each individual user, eliminating the need of broadcasting events to popular relays. 
 
-### Purposes
-
-The purpose of this NIP is to help clients find the events of the people they follow, to help tagged events get to the people tagged, and to help nostr scale better.
-
-### Suggestions
+## Final Considerations
 
-It is suggested that people spread their kind `10002` events to many relays, but write their normal feed-related events to a much smaller number of relays (between 2 to 6 relays). It is suggested that clients offer a way for users to spread their kind `10002` events to many more relays than they normally post to.
+1. Clients should guide users to keep `kind:10002` lists small (2-4 relays). 
```

Comment:
Not so sure about 2-4.  I think the low number should be 2, to imply redundancy.  Ideally the high number would be 4 as you say, but in the current environment it's safer to post to more relays so that clients that don't support NIP-65 are more likely to hear you.  I use 8 at the moment.